hass_nabucasa/account_link.py

```diff
@@ -32,25 +34,25 @@
 class AccountLinkException(Exception):
     """Base exception for when account link errors happen."""
 
-    def __init__(self, code: str):
+    def __init__(self, code: str) -> None:
         """Initialize the exception."""
         super().__init__(code)
-        self.code = code
+        self.code: str = code
 
 
-def _update_token_response(tokens, service):
+def _update_token_response(tokens: dict[str, str], service: str) -> None:
     """Update token response in place."""
     tokens["service"] = service
 
 
 class AuthorizeAccountHelper:
     """Class to help the user authorize a third party account with Home Assistant."""
 
-    def __init__(self, cloud: "Cloud", service: str):
+    def __init__(self, cloud: Cloud, service: str) -> None:
         """Initialize the authorize account helper."""
-        self.cloud = cloud
-        self.service = service
-        self._client: Optional[ClientWebSocketResponse] = None
+        self.cloud: Cloud = cloud
+        self.service: str = service
```

Comment:
These are already inferred from the method signature

```suggestion
        self.cloud = cloud
        self.service = service
```